packages/compiler/core/node-host.browser.ts

```diff
@@ -0,0 +1,5 @@
+export const NodeHost = undefined;
+
+// browsers shouldn't use this const and should instead
+// create some const at bundle time.
+export const cadlVersion = "browser";
```

Comment:
Would it be worthwhile to have this be generated in the build for both browser and not? Could file an issue if so.

Reply:
I think it would be nice to have that be the real value so we can show that in the playground but maybe as a separate issue.

Reply:
Definitely would be good to have but I'm not sure the best approach (would prefer just to import the package.json if possible, so bundlers can resolve it).

Reply:
Let's file an issue. It would be nice to gen the commit SHA too and show that in CLI and playground too.

Reply:
I can file it.

Reply:
https://github.com/microsoft/cadl/issues/311